locales/en/index.yml

```diff
@@ -183,6 +183,7 @@ profile:
         description: See the Privacy policy and the conditions of use of IO
         header: Information note on the processing of the IO app users's personal data
         updated: It is required to accept the new version of Terms and Conditions
```

Comment:
Since the Italian version has been updated, maybe this should become:
Abbiamo aggiornato la nostra Informativa e i Termini e condizioni d'uso
`We updated our Terms and conditions`